hw/ip/usbdev/doc/_index.md

```diff
@@ -144,6 +144,28 @@ To control the pullup on the D- line, this USB device features `dn_pullup_o` sig
 Of the two pullup signals `dp_pullup_o` and `dn_pullup_o`, only one can be enabled at any time.
 As this is a Full-Speed device, `dp_pullup_o`, i.e., the pullup on D+ is used by default (`pinflip` equals zero).
 
+### FPGA board implementation with PMOD
```

Comment:
Nit: I would capitalize the first letter of every word in this heading (to match the style with other headings in the doc)

Reply:
done